lnwallet/channel.go

```diff
@@ -849,6 +849,12 @@ func (lc *LightningChannel) diskCommitToMemCommit(isLocal bool,
 	diskCommit *channeldb.ChannelCommitment, localCommitPoint,
 	remoteCommitPoint *btcec.PublicKey) (*commitment, error) {
 
+	// If this commit is tweakless, then it'll affect the way we derive our
+	// keys, which will affect the commitment transaction reconstruction.
+	// So we'll determine this first, before we do anything else.
+	tweaklessCommit := (lc.channelState.ChanType ==
```

Comment:
parentheses can be elided, here and elsewhere

otoh we could define a similar `IsTweakless()` helper a la `IsSingleFunder()` to avoid code duplication

Reply:
Opted to make a `IsTweakless` method.